Content.Server/MassMedia/Systems/NewsSystem.cs

```diff
@@ -309,9 +309,6 @@ private void UpdateWriterDevices()
 
     private bool CanUse(EntityUid user, EntityUid console)
     {
-        // This shouldn't technically be possible because of BUI but don't trust client.
-        if (!_interaction.InRangeUnobstructed(console, user))
-            return false;
```

Comment:
This removes the check for everyone, not just AI.

Reply:
I don't understand why we even need this check. The comment says it "should't technically be possible". If I go from the console it just closes (watch attachment). Also haven't seen such check in other consoles' code.


https://github.com/user-attachments/assets/1955a62a-1b95-4f27-8061-ab4e40559cff

Reply:
BUI should be handling this, yes.